task-list/java/src/main/java/com/codurance/training/tasks/implementations/UncheckProject.java

```diff
@@ -0,0 +1,28 @@
+package com.codurance.training.tasks.implementations;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.lang.System.out;
+
+public class UncheckProject implements IUncheckProject {
+    public void setDone(String idString, boolean done, Map<String, List<Task>> tasks) {
+        String numericString = idString.replaceAll("\\D", "");
+        int id = Integer.parseInt(numericString);
+        for (Map.Entry<String, List<Task>> project : tasks.entrySet()) {
+            for (Task task : project.getValue()) {
+                if (task.getId() == id) {
+                    task.setDone(done);
+                    return;
+                }
+            }
+        }
+        out.printf("Could not find a task with an ID of %d.", id);
+        out.println();
+    }
```

Comment:
if the method is public and not included in interface then it would not be read
Better approach would be to make it private
Since you are using same functionality in CheckProject either you can have both check & uncheck in single class or 
Have setDone as Abstract method in parent